plugins/generator-1.21.1/neoforge-1.21.1/procedures/entity_set_attribute_base_value.java.ftl

```diff
@@ -0,0 +1,2 @@
+if (${input$entity} instanceof LivingEntity livingEntity${cbi})

```

Comment:
It may be smart to still do has attribute check here, as otherwise users will not do checks and report bugs about the game crashing due to NPEs

Reply:
the whole reason I added the has attribute procedure was to avoid this, this is a similar situation to entity procedure blocks which don't do null checks, is it really necessary to bloat the code to avoid some user's mistakes?

Reply:
Yes it is. I don't want to have another wiki page like this one: https://mcreator.net/wiki/my-entity-procedure-crashes-game

For entities, this is inevitable to a degree, but here we have quite a simple solution.

When adding features to the core, you need to keep the most beginner users in mind that never programmed before and don't really understand code at all.

The bloated code is cost of keeping the mod maker simple, which is the main purpose of MCreator.

Reply:
In other words, instead of blaming the users and asking ourselves

>  "why they don't know that, lol this is obvious"

we need to make it avoidable in the first place when possible